feat(payment-paypal): Add paypal payment method

JumpLink opened the pull request:
* Added PayPal icon
* Added PayPal translations
* Moved `PaymentSource` interfaces to common
* Updated `@beabee/beabee-common` to `0.20.3-alpha.2`
* Show PayPal Email address on account and admin contact overview pages

Related:
* https://github.com/beabee-communityrm/beabee/pull/384
* https://github.com/beabee-communityrm/beabee-common/pull/14

After merge:
* Merge the translations in the spread sheets branch tab

Preview:
![Bildschirmfoto vom 2024-03-26 17-39-01](https://github.com/beabee-communityrm/beabee-frontend/assets/1073989/34b828b3-7698-4a20-8684-aec19ba3fce6)

![image](https://github.com/beabee-communityrm/beabee-frontend/assets/1073989/c12c8250-1956-4ee5-b98f-0741b1c14aa3)

![image](https://github.com/beabee-communityrm/beabee-frontend/assets/1073989/36693b7e-1bb8-4062-9457-7c766cf4ce09)

![image](https://github.com/beabee-communityrm/beabee-frontend/assets/1073989/76759029-7833-4bf7-aeef-24514ab059f3)

JumpLink committed:
feat(feat/payment-paypal): Add PayPal icon and translations

JumpLink committed:
chore: Format code

wpf500 reviewed (changes requested):
Looks good so far. You will also need to update the `PaymentSource` component, which shows the user their current payment source on their contribution page. It should reflect the PayPal payment source you added to the backend

JumpLink committed:
feat(payment-paypal): Upgrade beabee-common

JumpLink committed:
feat(payment-paypal): Upgrade beabee-common to 0.20.3-alpha.2

JumpLink committed:
Merge branch 'main' into feat/payment-paypal

JumpLink committed:
feat(payment-paypal): Show paypal email address

JumpLink commented:
@wpf500 I have now moved `PaymentSource` to common because it's used from the frontend and the backend. For this, both projects must now use the latest common version, so I have also updated the frontend to `0.20.3-alpha.2`. There were a few places where typescript complained about the types, so I didn't invest too much time in that and made it easy for myself with the types for now.

I also added the paypal email beside the paypal icon on account and contact overview pages.

wpf500 commented on src/type/contribution-info.ts:
```diff
@@ -2,9 +2,10 @@ import type {
 } from '@beabee/beabee-common';
 
-import type { PaymentSource } from '@type';
+import type {} from '@type';
```
Delete this line

wpf500 reviewed (commented):
Good stuff. Just need to update the beabee-common version number once you've released v0.20.3 then this is good to go

JumpLink committed:
chore: Remove empty import

JumpLink committed:
chore: Upgrade @beabee/beabee-common to v0.20.3

JumpLink committed:
chore: Formatting
